tests/test_sunsch.py

```diff
@@ -355,8 +354,8 @@ def test_days_float(readonly_datadir):
     }
     sch = sunsch.process_sch_config(sunschconf)
     assert datetime.datetime(2020, 1, 11, 21, 36, 0) in sch.dates
-    # Rounding is downwards:
-    assert "11 'JAN' 2020/" in str(sch)
+    # TimeVector now support clocktimes
+    assert "11 'JAN' 2020 21:36" in str(sch)
```

Comment:
The Eclipse manual does not exemplify that 21:36 is an ok substitute for 21:36:00. Can you confirm that Eclipse will not choke on it?